arch/arm/soc/nxp_lpc/lpc54xxx/Kconfig.defconfig.lpc54114_m0

```diff
@@ -0,0 +1,42 @@
+# Kconfig - NXP LPC54114 M0 platform configuration options
+
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_LPC54114_M0
+
+config SOC
+	string
+	default lpc54114_m0
+
+config CPU_CORTEX_M_HAS_VTOR
```

Comment:
this should be a select under config SOC_LPC54114_M0

Reply:
Done.